### DISCUSSION TITLE
With single Milvus instance , Whether to  support querying two collections using multiple threads at the same time

### DISCUSSION THREAD

**Discussion by Haohao966** [milvus-io/milvus#3627] [Q&A and General discussion] [answered]
Have two collection ,  One is a dynamic library using FLAT index ,where new vectors can be added at any time, The other is the static collection using HNSW index, 3million, no new data  

Serial query of two collections is time-consuming,
Whether to  support querying two collections using multiple threads at the same time

**Comment by shiyu22** [comment, marked as answer]
You can certainly query two collections in multiple threads.

First of all, you need to set `cache_size` to be larger than the memory required for the two collections. Alternatively, you can start multiple Milvus clients and implement multiple threads to query the two collections.

**Reply by Haohao966** [reply]
thank you very much
